### PR TITLE
Awesome books-a complete website with navigation

### PR DESCRIPTION
**Pull request by elad237**
In this project, we created an Awesome books application and convert it into a fully functional website. 
We refactor the project to add three new sections on top of the two sections you have already completed: 
Books list (done), 
Add book form (done), 
a new Contact info section
, a Navigation bar, and a Footer.

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @elad237, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not contain a descriptive README.md file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by elad237** [comment]
> Hi @elad237,
> 
> While you made a great effort in this project, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that does not contain a descriptive README.md file.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

hello @FrederickMih there is a readme file

File: .README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This page registers books with author and title of the book. To register a book:
 2. Type the book's title.
 3. Click on Add button below.
 4. The books will be added to the list and will be displayed.
+5. added a navigation bar
 To remove a book from the list:
 1. Find the book in the list.
 2. Click on remove button for the respective book.
```
Comment (t-yanick) [inline .README.md:28]: - [OPTIONAL] To fix the issue with your readme file, please remove the dot before it's name, it should be `README.md` and not `.README.md`.

Comment (t-yanick) [inline .README.md:28]: - [OPTIONAL] To fix the issue with your readme file, please remove the dot before it's name, it should be `README.md` and not `.README.md`.

**Review by t-yanick — APPROVED** [review]
Hi @francis3400, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**